server/.env

```
@@ -1 +1 @@
-TEAM_MEMBERS="Aecio, Bonnie"
+TEAM_MEMBERS="Aecio, Bonnie, Shms"
```

```suggestion
TEAM_MEMBERS="Aecio, Bonnie, Shums"
```

hm, your name is Shums!